include/mbedtls/cipher.h

```diff
@@ -155,6 +161,12 @@ typedef enum {
     MBEDTLS_CIPHER_CAMELLIA_256_CCM,
 } mbedtls_cipher_type_t;
 
+#if !defined(MBEDTLS_DEPRECATED_REMOVED)
+typedef MBEDTLS_DEPRECATED mbedtls_cipher_type_t mbedtls_deprecated_cipher_type_t;
+#define MBEDTLS_CIPHER_ARC4_128 \
```

Comment:
It would be nice to make identifiers such as this one appear in the "Deprecated List" page of the Doxygen documentation. You can do that by adding a comment like
```
/** \deprecated ARC4 is deprecated and will likely be removed in a future version of the library. */
```
just before the `#define`.